.github/workflows_/.env

```diff
@@ -0,0 +1,10 @@
+BITNAMI_TEAM='["Akinorev","CeliaGMqrz","ClaaudiaGarcia","FraPazGal","Javirln","Mauraza","agarcia-oss","alemorcuq","alexherp","alvneiayu","andresbono","antgamdia","aoterolorenzo","beltran-rubo","bitnami-bot","carlossm","carrodher","castelblanque","corico44","cscazorla","dani8art","dariver","dcristobalhMad","dgomezleon","fevisera","fjagugar","fmulero","gdelgadot","gfichtenholt","gongomgra","javsalgar","jbianquetti-nami","jiparis","joancafom","jotadrilo","jotamartos","juamedgod","juan131","kaysavps","mdhont","migruiz4","mpermar","pablogalegoc","ppbaena","rafariossaa","rloporp","rogelio-o","tompizmor","xoanteis","zubero"]'
```

Comment:
Not used anywhere

Reply:
All the files in `.github/workflows_` are just a holdover from the Bitnami workflows.  By adding a `_` to that directory name, the workflows won't trigger.  Aspirationally, I'd like to adjust/adapt them for our use, but that's a chunk of work.

Reply:
I removed these and created a branch `workflows` that has them for consideration in the future.